packages/bootstrap/scss/card/_variables.scss

```diff
@@ -1,71 +1,184 @@
+/// @group card
 $kendo-card-footer-padding-x: $kendo-card-padding-x !default;
+/// The vertical padding of the Card footer.
+/// @group card
 $kendo-card-footer-padding-y: $kendo-card-padding-y !default;
+/// The top border width of the Card footer.
+/// @group card
 $kendo-card-footer-border-width: $kendo-card-border-width !default;
+/// The background color of the Card footer.
+/// @group card
 $kendo-card-footer-bg: $kendo-card-header-bg !default;
+/// The text color of the Card footer.
+/// @group card
 $kendo-card-footer-text: $kendo-card-header-text !default;
+/// The border color of the Card footer.
+/// @group card
 $kendo-card-footer-border: $kendo-card-header-border !default;
 
+/// The bottom margin of the Card subtitle.
+/// @group card
 $kendo-card-title-margin-bottom: k-map-get( $kendo-spacing, 3 ) !default;
+/// The text size of the Card title.
+/// @group card
 $kendo-card-title-font-size: $kendo-h5-font-size !default;
+/// The font family of the Card title.
+/// @group card
 $kendo-card-title-font-family: $headings-font-family !default;
+/// The line height of the Card title.
+/// @group card
 $kendo-card-title-line-height: $headings-line-height !default;
+/// The text weight of the Card title.
+/// @group card
 $kendo-card-title-font-weight: $headings-font-weight !default;
+/// The letter spacing of the Card title.
+/// @group card
 $kendo-card-title-letter-spacing: null !default;
 
+/// The bottom margin of the Card subtitle.
+/// @group card
 $kendo-card-subtitle-margin-bottom: k-map-get( $kendo-spacing, 3 ) !default;
+/// The font size of the Card subtitle.
+/// @group card
 $kendo-card-subtitle-font-size: $kendo-h6-font-size !default;
+/// The font family of the Card subtitle.
+/// @group card
 $kendo-card-subtitle-font-family: $headings-font-family !default;
+/// The line height of the Card subtitle.
+/// @group card
 $kendo-card-subtitle-line-height: $headings-line-height !default;
+/// The font weight of the Card subtitle.
+/// @group card
 $kendo-card-subtitle-font-weight: $headings-font-weight !default;
+/// The letter spacing of the Card subtitle.
+/// @group card
 $kendo-card-subtitle-letter-spacing: null !default;
+/// The text color of the Card subtitle.
+/// @group card
 $kendo-card-subtitle-text: $kendo-subtle-text !default;
 
+/// The maximum width of the Card image.
+/// @group card
 $kendo-card-img-max-width: 100px !default;
+/// The size of the avatar in the Card.
+/// @group card
 $kendo-card-avatar-size: 45px !default;
+/// The spacing between the avatar and the text in the Card.
+/// @group card
 $kendo-card-avatar-spacing: $kendo-card-header-padding-x !default;
 
+/// The horizontal padding of the Card actions.
+/// @group card
 $kendo-card-actions-padding-x: k-map-get( $kendo-spacing, 2 ) !default;
+/// The vertical padding of the Card actions.
+/// @group card
 $kendo-card-actions-padding-y: k-map-get( $kendo-spacing, 2 ) !default;
+/// The top border of the Card actions.
```

Comment:
Maybe here should be "The border width of the Card actions" or "The top border width of the Card actions" (for consistency)